app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -868,7 +868,12 @@ void OpenXRInputSource::Update(const XrFrameState& frameState, XrSpace localSpac
     delegate.SetHandActionEnabled(mIndex, isHandActionEnabled);
 
     device::CapabilityFlags flags = device::Orientation;
+#ifndef LYNX
```

Comment:
nit: Woudn't be clearer if we do "ifdef LYNX" ?

Reply:
Yes, the code would be a bit clearer with that change. Otherwise the change looks good.